### PR TITLE
Made profile page, added new routes in backed; delete request done

### PR DESCRIPTION
**Pull request by a5h101**
User can save products and see all his posts at profile page, made different components for cards. new routes for get request with nested filter added; completed the ui of userDetail page and added delete post functionality. Implemented logic for deleting post in front end and backend.

### CODE REVIEW

File: Rentit-backend/config.env
Comment (PatelNeelMahesh) [inline Rentit-backend/config.env:3]: You have pushed `config.env` file here. You should not keep it publicly available. Delete it from here and add file in `gitignore`.

**Review by PatelNeelMahesh — COMMENTED** [review]
Add `http://127.0.0.1:2000` in `.env` and everywhere in your codebase use the variable for domain.
Later when domain is changes you don't have to make changes in multiple files.

**Review by PatelNeelMahesh — CHANGES_REQUESTED** [review]
https://github.com/kalviumcommunity/RentIt/pull/1
https://github.com/kalviumcommunity/RentIt/pull/2
https://github.com/kalviumcommunity/RentIt/pull/3
https://github.com/kalviumcommunity/RentIt/pull/5
https://github.com/kalviumcommunity/RentIt/pull/6
https://github.com/kalviumcommunity/RentIt/pull/8
https://github.com/kalviumcommunity/RentIt/pull/9
https://github.com/kalviumcommunity/RentIt/pull/12
https://github.com/kalviumcommunity/RentIt/pull/13

For all above PR even after getting notified from Mentors you are merging pull request without getting approval, multiple time we have asked to pull domain name in `.env`. Aditya also informed in PR 8 

`Hey you should not merge the changes until it is approved. You have to make the changes and re-request to review it again.`

Again you are repeating the same behavior, considering that I am revoking your access to merge pull request.

Now you will be doing the pull request, after review if there are changes to make, once you complete those and commit the change, I will be merging pull request. 

Make sure you don't upload multiple files in a single PR, try raising PR with daily commits if possible.

Current PR is rejected since you have updated many features in it, raise at least 3 different PR for the current PR, keep the current comments in mind before raising the PR.

**Comment by anilgulecha** [comment]
@a5h101 is there a specific reason you're not heeding to this?

**Pull request closed without merging**
